e2e-tests/batch-mode-tests.sh

```diff
@@ -13,66 +12,31 @@ function print_message() {
   echo "${print_prefix} $*"
 }
 
-
 #######################################
-# Function that sets up testing env
+# Function that sets up e2e test
+# Arguments:
+#   Path where e2e-tests directory is. 
+#     Directory contains parquet tools jar
+#     as well as subdirectory of parquet file output
+#   Subdirectory to search for parquet files. 
+#      NON_JDBC or JDBC
+#   Optional: Flag to specify whether test is being 
+#      done on docker network.
```

Comment:
Please add a `Globals:` section too for the list of global variables that this uses ([style guide](https://google.github.io/styleguide/shellguide.html#function-comments)).